### DISCUSSION TITLE
Adding an offset to channel scanning

### DISCUSSION THREAD

**Discussion by Lagicrus** [ThioJoe/YT-Spammer-Purge#702] [Ideas]
Currently, you have to start from the first video in a channel in order to scan them for scams and spam.
But what happens if you scanned the first 10 and then want to scan the next 10? You have to copy and paste all 10 video links into the `specific videos` vs just telling it to scan the entire channel, starting at the 11th video.

What I am looking for is an ability to skip a definite amount of videos from a channel, from the latest, and then scan from there onwards.

**Comment by ThioJoe** [comment]
I see what you mean. I could probably add it in a future version, maybe the next. Not sure when that will be though
